tests/foreman/api/test_convert2rhel.py

```diff
@@ -267,7 +278,7 @@ def test_convert2rhel_oracle(target_sat, oracle, activation_key_rhel, version):
 
 
 @pytest.mark.e2e
-@pytest.mark.parametrize("version", ['centos7', 'centos8'], indirect=True)
+@pytest.mark.parametrize("version", ['centos7'], indirect=True)
```

Comment:
why removing centos8? and do we not test centos9 for C2R yet?
```suggestion
@pytest.mark.parametrize('version', ['centos7'], indirect=True)
```